src/admin-views/tribe-options-addons-api.php

```diff
@@ -7,6 +7,30 @@
 
 $current_url = Tribe__Settings::instance()->get_url( array( 'tab' => 'addons' ) );
 
+// if there's an Event Aggregator license key, add the Meetup.com API fields
+if ( get_option( 'pue_install_key_event_aggregator' ) ) {
+
+	$internal = array(
+		'meetup-start' => array(
+			'type' => 'html',
+			'html' => '<h3>Meetup</h3>',
```

Comment:
Why is this not translatable?

Reply:
Because "Meetup" is a brand name/trademark that should not be altered regardless of language.

Reply:
Nevermind, after a meeting today we have agreed this should be translatable after all. Will fix.